ci/prepare.jl

```diff
@@ -0,0 +1,12 @@
+# Show Julia information in verbose mode:
+versioninfo(true)
+
+# Initialize $JULIA_PKGDIR, in case this is the first time the test is run:
+Pkg.init()
```

Comment:
Otherwise:

```
$ time julia --color=yes -e 'Pkg.add("PyCall")'
ERROR: Package metadata directory /builds/tkfpub/pyjulia-testing-template/.julia/v0.6/METADATA doesn't exist; run Pkg.init() to initialize it.
```

https://gitlab.com/tkfpub/pyjulia-testing-template/-/jobs/68812116